curriculum/challenges/english/10-coding-interview-prep/project-euler/problem-63-powerful-digit-counts.md

```diff
@@ -10,37 +10,113 @@ dashedName: problem-63-powerful-digit-counts
 
 The 5-digit number, 16807 = 7<sup>5</sup>, is also a fifth power. Similarly, the 9-digit number, 134217728 = 8<sup>9</sup>, is a ninth power.
 
-How many `n`-digit positive integers exist which are also an `n`th power?
+Write function returning how many `n`-digit positive integers exist which are also an `n`th power.
```

Comment:
```suggestion
Write a function returning how many `n`-digit positive integers exist which are also an `n`th power.
```

A small tweak.

I'm not sure about the overall verbiage of this line - it feels clunky to me. But if we fix the grammar error with this suggestion, I can live with it.

Reply:
It does sound clunky, and takes a while to narrow what exactly is wanted. I'm not sure if this isn't part of the original challenge, to make it a bit harder. If order is reversed - How many `n`th powers of a number is also a `n`-digit positive integer - it is much easier to approach.

Reply:
What about: `Complete the function so that it returns how many positive integers are of length `n` and an `n`th power.`

Reply:
> It does sound clunky, and takes a while to narrow what exactly is wanted.

I think that is what is nice about Project Euler problems.  Everyone gets the same verbiage of the challenge.  I personally don't like changing the original wording of these problems or the scope of the problems.  Just my opinion.